docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +97,45 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
-decorator. The decorator takes in a function resolving the target model class
-constructor and designates the relation type. It also calls `property()` to
-ensure that the decorated property is correctly defined.
+The standard naming convention for the foreign key property in the source model
+is `target model name` + `Id` (for example, Order.customerId).
+
+- If the foreign key property name in the source model has to be customized,
+  then the `name` attribute in the relation definition (the target model name)
+  has to be stated explicitly. Also, the corresponding `hasMany` relation in the
+  `target` model has to be modified with the correct `keyTo` value.
+- In the following example, the foreign key property name is customized as
+  `cust_Id` instead of `customerId`. so the relation definition in the second
+  argument is explicitly passed to the `belongsTo` decorator.
+
+```ts
+class Order extends Entity {
+  // constructor, properties, etc.
+  @belongsTo(() => Customer, {keyFrom: 'cust_Id', name: 'customer'})
+  cust_Id: number;
+}
+```
+
+In the following example, the db column name of the foreign key is customized by
+passing the property definition in the third argument to the `belongsTo`
+decorator.
+
+```ts
+class Order extends Entity {
+  // constructor, properties, etc.
+  @belongsTo(() => Customer, {keyFrom: 'customerId'}, {name: 'customer_id'})
+  customerId: number;
+}
+```
 
-A usage of the decorator with a custom primary key of the target model for the
-above example is as follows:
+The standard naming convention for the primary key in the target model is `id`.
```

Comment:
Same as above, `keyTo` has a default value (not just a naming convention), which is the primary key of the source model. 
By default, no matter what the name of the id property is in the source mode, `keyTo` would set to be that name automatically. In you case, the default would be `customized_target_property` directly. [See file](https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/relations/belongs-to/belongs-to.helpers.ts#L60).

If the primary key we use in this `belongsTo` relation is still be the id property of source model, `keyTo` should be optional here. 

We probably had `id` as a default value before. So it was documented this way. I'd suggest to remove this example or modify it to suit our code base more.

Reply:
agreed